composer.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "WP-API/WP-API",
+  "description":"JSON-based REST API for WordPress, developed as part of GSoC 2013.",
+  "keywords":["search"],
+  "authors":[
+	{
+	  "name":"Ryan McCue",
+	  "homepage":"http://ryanmccue.info/"
+	}
+  ],
+  "type": "wordpress-plugin",
+  "require": {
+    "composer/installers": "v1.0.6",
+    "php": ">=5.3.2"
```

Comment:
The version of PHP used for composer can differ from the version used for WP-API.

For example a user might have 5.2.4 installed on her server, but access a newer version of PHP from the command line for composer, e.g., `php54 composer.phar install`

I think the PHP requirement in composer.json should match the version of PHP that's actually required by WP-API.